src/batch-client/index.ts

```diff
@@ -169,6 +169,49 @@ function normalizeMessage(
 	);
 }
 
+const hasUnreadDescendent = (folder: any) => {
+	if (
+		folder[
+			folder.name === 'Drafts' || folder.name === 'Outbox'
```

Comment:
instead of checking folder names, can we check folder ids?

Reply:
I couldnt find a folderId constant for Outbox so wll keep this, name only